fili-core/src/main/java/com/yahoo/bard/webservice/table/availability/MetricUnionAvailability.java

```diff
@@ -58,7 +60,7 @@
  * +----------------------+------------------+
  * |  [metric2]           |  [2016/2017-03]  |
  * +----------------------+------------------+
- * |  [metric1, metric2]  | [2017/2018]      |
+ * |  [metric1, metric2]  | [2017/2017-03]   |
```

Comment:
@michael-mclawhorn Please confirm this doc change to make sure it makes sense to you, Thanks~